src/openforms/authentication/tests/test_views.py

```diff
@@ -384,3 +391,179 @@ def test_co_sign_flow_invalid_submission_id(self):
 
         self.submission.refresh_from_db()
         self.assertEqual(self.submission.co_sign_data, {})
+
+
+@override_settings(
+    CORS_ALLOW_ALL_ORIGINS=False, CORS_ALLOWED_ORIGINS=["http://testserver"]
+)
+class RegistratorSubjectInfoViewTests(WebTest):
+    csrf_checks = False
+
+    def setUp(self):
+        super().setUp()
+
+        self.form = FormStepFactory(
+            form__slug="myform",
+            form__authentication_backends=["plugin1"],
+            form_definition__login_required=True,
+        ).form
+
+        form_path = reverse("core:form-detail", kwargs={"slug": self.form.slug})
+        self.form_url = f"http://testserver{form_path}"
+
+        f = furl(
+            reverse(
+                "authentication:registrator-subject",
+                kwargs={"slug": self.form.slug},
+            )
+        )
+        self.subject_url_missing_next = f.url
+
+        f.args["next"] = "http://not-in-whitelist.net/foo"
+        self.subject_url_not_allowed_next = f.url
+
+        f.args["next"] = self.form_url
+        self.subject_url = f.url
+
+        self.user = UserFactory(
+            user_permissions=["of_authentication.can_register_client_submission"]
+        )
```

Comment:
perhaps put this in `setUpTestData` instead to keep the tests as fast as possible :)